### PR TITLE
modal blocked

### PR DESCRIPTION
**Pull request by juangdelvalle**
Set a presentationController delegate to be able to capture the event when the presented view is dismissed by a swap down with the new modality management of iOS 13, implementing the presentationControllerDidDismiss method

### CODE REVIEW

File: Yoshi.podspec
Comment (MichaelCampbell) [inline Yoshi.podspec]: Is the description no longer required — curious why you removed it?

**Comment by juangdelvalle** [comment]
@paulmiard @MichaelCampbell I couldn't add you as a reviewer on the PR, so if you can add it to your list of pending reviews to check it when you have a chance it would be great. Thanks on advance!

File: Yoshi.podspec
Comment (MichaelCampbell) [inline Yoshi.podspec:29]: Is platform no longer required?

File: Yoshi.podspec
Comment (MichaelCampbell) [inline Yoshi.podspec:13]: Should we no longer link reference the license file?

**Review by MichaelCampbell — APPROVED** [review]
This looks great, thanks for working on it @juangdelvalle. Sorry for the delay on reviewing.

I left some inline comments, but I think we should be good to merge once the travis build passes. Let me know if you need some support getting it to pass.

File: Yoshi.podspec
Comment (paulmiard) [inline Yoshi.podspec]: There is an open PR that takes care of Swift 5 changes. Don't we need to merge it before adding 5.x to the list?
See https://github.com/prolificinteractive/Yoshi/pull/83

File: Yoshi.podspec
Comment (juangdelvalle) [reply Yoshi.podspec:13]: as I understand, is no longer required to make a reference to a license file, but I can change it to make the reference anyway.

File: Yoshi.podspec
Comment (juangdelvalle) [reply Yoshi.podspec]: it was by mistake, I'll add it back right away

File: Yoshi.podspec
Comment (juangdelvalle) [reply Yoshi.podspec:29]: platform is now specified in  `s.ios.deployment_target = '8.0'`

File: Yoshi.podspec
Comment (juangdelvalle) [reply Yoshi.podspec]: In this PR I also migrated the code to Swift 5 (the other PR wasn't approved yet so I thought about killing two birds with the same stone)

**Comment by juangdelvalle** [comment]
> This looks great, thanks for working on it @juangdelvalle. Sorry for the delay on reviewing.
> 
> I left some inline comments, but I think we should be good to merge once the travis build passes. Let me know if you need some support getting it to pass.

@MichaelCampbell I might need help to get it to pass, the error that I'm getting is: `SWIFT_VERSION '5.0' is unsupported, supported versions are: 3.0, 4.0, 4.2. (in target 'YoshiTests')` I wonder if the supported swift version is something that must be setted on Travis

File: Yoshi.podspec
Comment (paulmiard) [reply Yoshi.podspec]: I didn't look at the details in the other PR but looked like there was more going on. Are you saying the other PR can be declined all together once this is merged? I guess I will diff it to see what else was in there once we merge but happy to hear your thoughts

**Comment by MichaelCampbell** [comment]
> @MichaelCampbell I might need help to get it to pass, the error that I'm getting is: `SWIFT_VERSION '5.0' is unsupported, supported versions are: 3.0, 4.0, 4.2. (in target 'YoshiTests')` I wonder if the supported swift version is something that must be setted on Travis

I took a look into this framework and it seems that they're doing something similar to us: https://github.com/DianQK/TransitionTreasury/blob/master/.travis.yml

Maybe try bumping the pod version to latest @juangdelvalle

**Comment by Armenm** [comment]
Hi @juangdelvalle, any progress on this PR?

**Pull request closed without merging**
